### ISSUE TITLE
[typescript-angular] Generation on Windows creates paths with wrong file separator

### ISSUE THREAD

**Issue by wtho** [swagger-api/swagger-codegen-generators#335]
## Issue
When generating (v3.0.5) using
```bash
java -jar \
  -DsupportsES6=true \
  swagger-codegen.jar generate \
  -l typescript-angular \
  -i schema.json \
  -o src/app/generated/
```
on Windows wrong file separators are used inside the TypeScript files.

This creates problems when transpiling TypeScript.

On Linux everything works fine.

## Expected Output
`api.service.ts`
```ts
[...]
import { SomeRequest } from '../model/someRequest';
[...]
```

## Actual Output
`api.service.ts`
```ts
[...]
import { SomeRequest } from '../model\someRequest';
[...]
```

**Comment by fredspies** [comment]
The issue persists in v3.08.

**Comment by HugoMario** [comment]
fixed by #526
